lib/api/exception.dart

```diff
@@ -131,3 +131,5 @@ class MalformedServerResponseException extends ServerException {
             .errorMalformedResponseWithCause(
               httpStatus, causeException.toString()));
 }
+
+class AccountAlreadyExistsException implements Exception {}
```

Comment:
This `lib/api/` tree is for bindings to the Zulip server API. So this exception should go somewhere else.

Probably `lib/model/store.dart` (where it gets thrown from), at the bottom of the file, is good.